### PR TITLE
Remove `enableImmutableInstalls: false`

### LINKED ISSUE #35: Should immutable installs be enabled by default?
Currently these are disabled:

https://github.com/jupyterlab/extension-template/blob/f256e361c83e9c3143c96c2544c62de7c2cf1f21/template/.yarnrc.yml#L1

As `jlpm` on local machine and on CI matches (i.e. JupyterLab version is the same) there should be no issues. If needed users can disable it downstream. I know it is hard when trying to support multiple versions or with very complex dependencies, but maybe worth enabling by default anyways?

### PR DESCRIPTION
**Pull request by fcollonval**
Fixes #35

Remove yarn option to be more strict on CI.

### CODE REVIEW

**Comment by fcollonval** [comment]
CI will be broken (fix is part of #38)

**Comment by jtpio** [comment]
Rebasing to kick CI.

File: template/{% if test %}ui-tests{% endif %}/tests/{{python_name}}.spec.ts.jinja [hunk 1]
```diff
@@ -32,6 +32,9 @@ test('should emit an activation console message', async ({ page }) => {
 
   await page.getByRole('button', { name: 'Rename' }).click();
 
+  // Close file opened as editor
+  await page.activity.closePanel('test.my_type');
```
Comment (fcollonval) [inline template/{% if test %}ui-tests{% endif %}/tests/{{python_name}}.spec.ts.jinja:36]: @jtpio @krassowski I would like your input on this.

With 4.0.5, the `page.filebrowser.open(filename)` code now fails if a file is opened multiple times (different viewers) as in this case. My question, should it be fixed per test as here or fix upstream by allowing to find any number of panel opened for a given filename?

Comment (jtpio) [reply template/{% if test %}ui-tests{% endif %}/tests/{{python_name}}.spec.ts.jinja:36]: > (different viewers) as in this case. My question, should it be fixed per test as here or fix upstream by allowing to find any number of panel opened for a given filename?

I would say upstream ideally? Could this also fix https://github.com/jupyterlab/jupyterlab/issues/13188 maybe?

Comment (fcollonval) [reply template/{% if test %}ui-tests{% endif %}/tests/{{python_name}}.spec.ts.jinja:36]: I opened https://github.com/jupyterlab/jupyterlab/pull/14983

**Pull request merged**